.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "extends": ["eslint:recommended"],
```

Comment:
replace recommended to lint from chronobank - please, have a look at bitcoin middleware repo (we already have such stuff there)